designdocs/encoding.md

```diff
@@ -0,0 +1,131 @@
+# Encodings
```

Comment:
Can we add a section on what kind of data we want to encode and how we plan to use the data?